### PR TITLE
Cluster setup

### PR DESCRIPTION
**Pull request by mohit-surana**
- Modified matplotlib invocation to save plots instead of opening interactively (which is not possible over ssh)
- Added debug statements to track progress
- Added instructions to setup the code base on the swarm cluster

### CODE REVIEW

File: cluster-setup.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+$ scp data/all_emails.csv s18<username>@swarm2.cs.umass.edu:/home/s18<username>/projects/organizational-roles/data
+
+$ pip install --user http://download.pytorch.org/whl/cpu/torch-0.3.1-cp27-cp27mu-linux_x86_64.whl
+$ pip install --user torchvision 
+$ pip install --user gensim
+$ pip install --user MySQL-python
+$ pip install --user mpld3
+$ pip install --user seaborn
+$ pip install --user nltk
+
```
Comment (mudit1990) [inline cluster-setup.txt:17]: awesome! 🥇

File: code/dal.py [hunk 2]
```diff
@@ -19,16 +19,19 @@ def get_emails(num_emails=100, max_users=150, fetch_all=False):
...
         data = np.loadtxt(file_path + file_name, dtype='str', delimiter=',')
     else:
         file_name = __get_appr_filename(num_emails, file_path)
         data = np.loadtxt(file_path + file_name, dtype='str', delimiter=',')
     
+    print 'Loaded data. Filtering by users'
     data = __filter_mails_by_users(data, max_users)
     if not fetch_all and len(data) > num_emails:
         data = data[:num_emails]
+    print 'Data filtered by users'
```
Comment (mudit1990) [inline code/dal.py:34]: okay, these print statements will also need to be present in the new methods. can you make sure they are there when I raise a pull request

**Review by mudit1990 — APPROVED** [review]
looks good ... okay to merge

File: code/dal.py (hunk 2 above)
Comment (mohit-surana) [reply code/dal.py:34]: Will keep this in mind

**Pull request merged**